blazor/scheduler/header-rows.md

```diff
@@ -90,6 +92,8 @@ To display the timeline Scheduler simply with year and month names alone, define
 }
 ```
 
+![Display year and month rows in timeline views](images/blazor-scheduler-timelineheaderrows-monthandyear.png)
```

Comment:
Add platform name and component name in image's alt text to avoid duplicates.

"Display year and month rows in Blazor Scheduler timeline views"